Jenkinsfile

```diff
@@ -1,3 +1,12 @@
+#!/usr/bin/env groovy
+
+def excludePatterns = [
```

Comment:
I would expect patterns to work with either regex or glob not static strings as currently `.md` is very ambiguous.